pkg/pwgen/pwgen_test.go

```diff
@@ -93,7 +93,12 @@ func TestGeneratePasswordWithAllClasses(t *testing.T) {
 }
 
 func TestGenerateMemorablePassword(t *testing.T) {
-	pw := GenerateMemorablePassword(20, false)
+	pw := GenerateMemorablePassword(20, false, false)
+	assert.GreaterOrEqual(t, len(pw), 20)
+}
+
+func TestGenerateMemorablePasswordCapital(t *testing.T) {
+	pw := GenerateMemorablePassword(20, false, true)
 	assert.GreaterOrEqual(t, len(pw), 20)
```

Comment:
It would be nice to assert it contains a uppercase. You can compare the string with the lowercase version `strings.ToLower(pw)` as it shouldn't be the same in that case.

Also maybe add in the previous test that it is only lowercase if set to `false`?